src/OpenLoco/Company.cpp

```diff
@@ -343,7 +343,7 @@ namespace OpenLoco
         { CorporateRating::tycoon, StringIds::corporate_rating_tycoon },
     };
 
-    static string_id getCorporateRatingAsStringId(CorporateRating rating)
```

Comment:
nice, I think less `static` methods is a good thing (right?)

Reply:
well static methods just mean they are only used in the one file. It ultimately doesn't make much difference if a method is static or not just means slightly more of the code is joined up.